File: src/test/data/rename_casualties/AfterMethodRename.java
public class MyClass {
    private int x, y;

    public MyClass() {
        this.x = 0;
        this.y = 0;
    }

    public int getSum() {
        return readX() + readY();
    }

    private int readX() {
        return this.x;
    }

    private int readY() {
        return this.y;
    }
}

File: data/rename_casualties/AfterMethodRename.cpp
class MyClass {
private:
    int x;
    int y;

    int readX() const {
        return x;
    }

    int readY() const {
        return y;
    }

public:
    MyClass() : x(0), y(0) {}

    int getSum() const {
        return readX() + readY();
    }
};
